.github/workflows/continuous-integration.yml

```diff
@@ -5,28 +5,27 @@ name: Continuous Integration
 
 on:
   push:
-    branches: 
+    branches:
       - '**'
-      
+
   pull_request:
-    branches: 
+    branches:
       - '**'
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         node-version: [14.x]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build 
-    - run: npm test
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install --frozen-lockfile
+      - run: yarn build
+      - run: yarn test
```

Comment:
Why was this change made?

Reply:
These changes were made by the prettier extension that I have installed on vscode.